### ISSUE TITLE
Public ILL request form: better validation management

### ISSUE THREAD

**Issue by PascalRepond** [rero/rero-ils#3725]
### How it works

When one fills in an ILL request from the public UI, there are modals:

1. To warn that the document is present in my organisation (conditionnal)
2. To warn that the request may include fees (always)

These are displayed even when the form is not valid. Only after are the probelmatic fields highlighted. This might lead the user to believe the form has been sent correctly. 

### Improvement suggestion

To be decided based on how this form works.

- On clicking "save", validate the form (wihtout saving it) before showing the modals. Show modals only if the form is valid and ready to be sent.
- OR Disable the "Save" button if the form is not valid.
- Nice-to-have: Add an alert message if the form is not valid.
